functions/src/export-csv.ts

```diff
@@ -96,43 +93,37 @@ export async function exportCsvHandler(
 
   lois.forEach(loi => {
     const loiId = loi.id;
-    const location = loi.get('location') || {};
     const submissions = submissionsByLocationOfInterest[loiId] || [{}];
     submissions.forEach(submission => {
       const row = [];
-      row.push(loi.get('id') || '');
-      row.push(location['_latitude'] || '');
-      row.push(location['_longitude'] || '');
-      row.push(toWkt(loi.get('geoJson')) || '');
+      // Header: fid
+      row.push(loi.get('properties').id || '');
+      // Header: geometry
+      row.push(toWkt(loi.get('geometry')) || '');
       // TODO(#1288): Clean up remaining references to old responses field
-      const data = submission['data'] || submission['responses'] || submission['results'] || {};
+      const data =
+        submission['data'] ||
+        submission['responses'] ||
+        submission['results'] ||
+        {};
+      // Header: loop over survey
       tasks.forEach((task, taskId) => row.push(getValue(taskId, task, data)));
       csvStream.write(row);
     });
   });
   csvStream.end();
 }
 
-function toWkt(geoJsonString: string) {
-  const geoJsonObject = parseGeoJson(geoJsonString);
-  const geometry = getGeometry(geoJsonObject);
-  return geometry ? geojsonToWKT(geometry) : '';
-}
-
-function parseGeoJson(jsonString: string) {
-  try {
-    // Note: Returns null when jsonString is null.
-    return JSON.parse(jsonString);
-  } catch (e) {
-    return null;
+function toWkt(geometryObject: any): string {
+  // Make sure that the coordinates field is no longer using the Firestore format
```

Comment:
By "Firestore format" I assume you mean the representation we use for geometries in Firestore? Would that only happen because of an internal bug? If so, we should probably throw an error to fail fast rather than to swallow/hide the issue.

Reply:
Yes that's the representation I'm talking about.

I'm not super familiar with error handling in this case, so I am using some intution let me know your thoughts.

Reply:
If this is a failure that "should never happen" we should throw an error.